core/persistence-jpa-json/src/test/resources/domains/MasterContent.xml

```diff
@@ -2409,59 +2409,4 @@ $$ }&#10;
   <SecurityQuestion id="887028ea-66fc-41e7-b397-620d7ea6dfbb" content="What's your mother's maiden name?"/>
 
   <SyncopeLogger logName="syncope.audit.[LOGIC]:[SyncopeLogic]:[]:[isSelfRegAllowed]:[SUCCESS]" logLevel="DEBUG" logType="AUDIT"/>
-
```

Comment:
You removed all audit entries, not just read / list / search as I was suggesting... any reason why?

Reply:
Is `AuditITCase` working now, for example?

Reply:
ah, sorry my bad. Read your comment a little too quickly! I will clean this up to only remove those entries.

Reply:
Also I think I'd need to also include the same batch of audit loggers for AnyObjects and add test cases for those too.

Reply:
Ok; don't forget to align this file with `core/persistence-jpa-json/src/test/resources/domains/MasterContent.xml`